ios/emurgo.xcodeproj/project.pbxproj

```diff
@@ -977,6 +1056,8 @@
 				2D16E6871FA4F8E400B85C8A /* Frameworks */,
 				F7FBD967219C5A6D003FE9E2 /* Recovered References */,
 				661CFB1B626FF527BA54DEF4 /* Pods */,
+				E9A1C61824183FB700151CFA /* emurgo copy2-Info.plist */,
+				E9A1C80B24185CCA00151CFA /* emurgo copy3-Info.plist */,
```

Comment:
@v-almonacid I’m not sure if it’s unused or just stuff that didn’t get rename 🤔

Reply:
@nicarq I believe that the `emurgo copyX.plist` files were created when new targets were added. When I created the nightly target I noticed `emurgo copy3.plist` was created. You can check which plist file is used by each target in `target > Build Settings > Info.plist File`.
So it should be:
- emurgo -> emurgo/info.plist
- emurgo-staging -> Staging-info.plist
- emurgo-nightly -> Nightly-Info.plist

I assumed it was safe to remove the other ones, and so I did in the last commit. But I didn't realise there were still some references in the project. I don't think they are used though.